fairlearn/metrics/_input_manipulations.py

```diff
@@ -17,3 +17,16 @@ def _convert_to_ndarray_and_squeeze(target):
         result = result.reshape(1)
 
     return result
+
+
+def _convert_to_ndarray_1d(target):
+    """Convert the input to a `numpy.ndarray` with a single dimension.
+
+    If the input is multi-dimension, this will raise an exception
+    """
+    result = _convert_to_ndarray_and_squeeze(target)
+
+    if len(result.shape) > 1:
+        raise ValueError("Input not 1D")
```

Comment:
can we make this string a constant and import it in the test? Otherwise we'll have to keep two strings in two places consistent

Reply:
I've made the string a more descriptive constant. I'm not referencing from the test, since I think that's the sort of change which should trigger a test update.